common/djangoapps/course_modes/tests/test_models.py

```diff
@@ -471,3 +471,22 @@ def test_verified_min_price(self, mode_slug, is_error_expected):
             self.assertTrue(is_error_expected, "Did not expect a ValidationError to be thrown.")
         else:
             self.assertFalse(is_error_expected, "Expected a ValidationError to be thrown.")
+
+
+class EnrollmentModeTest(TestCase):
+    """
+    Tests for EnrollmentMode Model.
+    """
+    def tearDown(self):
+        super(EnrollmentModeTest, self).tearDown()
+        EnrollmentMode.objects.all().delete()
```

Comment:
@jzoldak is this true? There are a ton of unit tests that delete models in the tearDown. Are all of those unnecessary?